utilities/check_account/checks/check_account_spec.rb

```diff
@@ -38,8 +41,10 @@
       expect(iam_utils.get_users.find { |x| x[:has_password] }).to be_nil
     end
 
-    it "should not have any accesses keys greater than #{MAX_KEY_AGE} days old" do
-      expect(iam_utils.get_active_keys_older_than_n_days(MAX_KEY_AGE)).to be_empty
+    it "any access keys older than #{MAX_KEY_AGE} days should be whitelisted" do
+      iam_utils.get_active_keys_older_than_n_days(MAX_KEY_AGE).each do |key|
+        expect(IAM_KEY_WHITELIST).to include(key[:base_data].user_name)
+      end
```

Comment:
Also this is more cleanly done
```
it "any access keys older than #{MAX_KEY_AGE} days should be whitelisted" do
  older = iam_utils.get_active_keys_older_than_n_days(MAX_KEY_AGE)
  expect(older - whitelist).to be_empty
end
```

Reply:
The approach in the PR has the side effect of including the content of the whitelist array in the rspec output and the non-whitelisted / failing username. I thought it would be more useful output for the user than dumping the struct, which is what this change does:

```
  1) Account configuration check Check IAM Policies any access keys older than 90 days should be whitelisted
     Failure/Error: expect(older - whitelist).to be_empty
       expected `[{:base_data=>#<struct Aws::IAM::Types::AccessKeyMetadata user_name="REDACTED...LNHPRMPXFJA", status="Active", create_date=2016-08-25 02:35:48 UTC>, :active=>true, :days_old=>XXX}].empty?` to return true, got false
     # ./utilities/check_account/checks/check_account_spec.rb:48:in `block (3 levels) in <top (required)>'
```

Reply:
Yeah I neglected to fetch the key within the result of `get_active_keys_older_than_n_days`, sorry about that.